tests/api/test_wisdem_api.py

```diff
@@ -1,47 +0,0 @@
-"""Tests for the Monopile Wisdem API."""
-
-__author__ = ["Jake Nunemaker"]
-__copyright__ = "Copyright 2020, National Renewable Energy Laboratory"
-__maintainer__ = "Jake Nunemaker"
-__email__ = "jake.nunemaker@nrel.gov"
-
-
-import openmdao.api as om
-
-from ORBIT.api.wisdem import Orbit
```

Comment:
Should this API file also be dropped?

Reply:
Yeah, this file was removed in a follow-up commit.

Reply:
FYI, I'm still seeing `api/wisdem.py` in the `noomdao_newpythons` branch. I think I might be missing something, or you may have not committed the file removal change?

Reply:
Yup- I had removed the test but not the file itself.  Fixed now!

Reply:
Thank you!!